k8s/deployment.yaml

```diff
@@ -15,6 +15,7 @@ spec:
       labels:
         app: kubeportal-api-dev
     spec:
+      priorityClassName: important
```

Comment:
Why?

Reply:
I wanted to give it a try if this is the reason why CICD doesn't work. (it's kind of this student-mood: 'I try things and I don't care if that makes sense'. Sorry, I know.).  I also updated the runner. I don't know what's wrong.

Reply:
I also wanted to talk with you about this priorityClass-stuff in the next days.